explanation/applications.md

```diff
@@ -9,6 +9,20 @@ To run on the Anbox Cloud platform, applications must fulfil the following requi
 
 If your application fulfils these requirements but you are still having issues running it on Anbox Cloud, file a [bug report](https://bugs.launchpad.net/indore-extern/+filebug).
 
+## Possible application status
+
+The following table lists the different status that an application can have depending on its state and what each status means:
+
+| Application status | Description |
+|--------------------|-------------|
+| `initializing` | The application is being created. Wait until the application reaches the `ready` state. |
+| `ready` | The application is successfully created and ready to be used. |
+| `deleted` | The application is deleted and no longer available for use. |
```

Comment:
Should we use the past tense instead? e.g.,
The application was deleted and is no longer available for use.

I'm not sure what's better, wdyt?